### ISSUE TITLE
alert_job container running as curl_user

### ISSUE THREAD

**Issue by dzielinski4895** [robusta-dev/robusta#1208]
Is there anyway to configure this to run as a UID to satisfy running as a nonroot user?

Running robusta demo-alert --namespaces monitoring generates the alert-job, but due to policy, the container will not start because it cannot verify user is nonroot.

**Comment by arikalon1** [comment]
hi @dzielinski4895 

Can you share which configuration part in the job manifest is blocked by your policy?

**Comment by dzielinski4895** [comment]
There is no configuration to have the job pod/container run as a specific UID. So spec.template.spec.containers.securityContext.runAsUser

**Comment by arikalon1** [comment]
fixed, will be included in 0.10.27
If you want to use it sooner, you can build the cli locally:
```
poetry install -E all
poetry build && pip3.9 install dist/robusta_cli-0.0.0-py3-none-any.whl --force-reinstall --no-cache
```
